setup.cfg

```diff
@@ -11,7 +11,7 @@ exclude =
 
 [tool:pytest]
 addopts = --flake8 --black
-flake8-ignore = 
+flake8-ignore =
   *.py E133 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503 W504
   bin.src/*.py ALL
```

Comment:
Please delete this line; there is no such directory

Reply:
Done.